packages/libzip/build.sh

```diff
@@ -0,0 +1,10 @@
+TERMUX_PKG_HOMEPAGE=https://libzip.org/
+TERMUX_PKG_DESCRIPTION="library for reading, creating, and modifying zip archives."
+TERMUX_PKG_VERSION=1.5.1
+TERMUX_PKG_REVISION=1
```

Comment:
Not necessary.

Reply:
Ok thanks fixed in https://github.com/termux/termux-packages/pull/3433/commits/1a5b7fbaa7517efec2696bc9a16db0daeb4c2b13